pages/print_labels/reagent_aliquots.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <b-container>
+    <b-row>
+      <b-col>
+        <PrintLabelsRouter />
+        <h1>Print reagent aliquot labels</h1>
+        <p class="lead"></p>
+
+        <!-- TODO: better in a component of its own? -->
+        <p>
+          <b-alert :show="isError" dismissible variant="danger">
+            {{ alertMessage }}
+          </b-alert>
+          <b-alert :show="isSuccess" dismissible variant="success">
+            {{ alertMessage }}
+          </b-alert>
+          <b-alert :show="isBusy" dismissible variant="warning">
+            {{ alertMessage }}
+          </b-alert>
+        </p>
+        <p>
+          <img src="@/assets/images/reagent_aliquot_label_preview.png" />
+        </p>
+        <p>
+          <label for="selectPrinter">Which printer would you like to use?</label>
+          <b-form-select id="selectPrinter" v-model="printer" :options="printers"></b-form-select>
+        </p>
+        <p>
+          <label for="firstLineText">Freeform first line of text:</label>
+          <b-form-input id="firstLineText" v-model="firstLineText" type="text"></b-form-input>
+        </p>
+        <p>
+          <label for="secondLineText">Freeform second line of text:</label>
+          <b-form-input id="secondLineText" v-model="secondLineText" type="text"></b-form-input>
+        </p>
+        <p>
+          <label for="barcode">Scan or enter an aliquot barcode:</label>
+          <b-form-input id="barcode" v-model="barcode" type="text"></b-form-input>
+        </p>
+        <p>
+          <label for="numberOfLabels">Quantity of labels needed:</label>
+          <b-form-input
+            id="numberOfLabels"
+            v-model="numberOfLabelsString"
+            type="number"
+            value="1"
+            min="1"
+          ></b-form-input>
+        </p>
+        <p class="text-right">
+          <b-button
+            id="printLabels"
+            block
+            size="lg"
+            variant="success"
+            :disabled="isBusy || !isValid"
+            @click="printLabels"
+          >
+            Print labels
+            <b-spinner v-show="isBusy" small></b-spinner>
+          </b-button>
+        </p>
+      </b-col>
+    </b-row>
+  </b-container>
+</template>
+
+<script>
+import statuses from '@/modules/statuses'
+import PrintLabels from '@/modules/sprint_reagent_aliquot_labels'
+import config from '@/nuxt.config'
+import PrintLabelsRouter from '@/components/PrintLabelsRouter'
+
+export default {
+  components: {
+    PrintLabelsRouter,
+  },
+  props: {
+    printers: {
+      type: Array,
+      default() {
+        return config.publicRuntimeConfig.printers.split(',')
+      },
+    },
+  },
+  data() {
+    return {
+      status: statuses.Idle,
+      alertMessage: '',
+      printer: 'heron-bc1',
```

Comment:
is heron-bc1 an initial value for a reason? What if it doesnt exist in a particular env

Reply:
Fair point.  I'm not sure.  This is just what the other label printing pages already do.  I'll have a think about what we should do here.  Probably have an empty string since the component doesn't auto populate with this value on dev.  I think it might be worthwhile make the form invalid without a printer selection as well.